### DISCUSSION TITLE
[2024-10-31] APAC and EMEA Building DevRel Foundation Community Call

### DISCUSSION THREAD

**Discussion by anajsana** [DevRel-Foundation/governance#115] [Meeting Notes]
> Part of our community calls are being used as Open Office Hours to reach community consensus on Foundation Next Steps. If you would like to get involved, this is a safe and welcoming space to start!

> Whether you are new to the call or a regular participant, you can add your discussion topics as a comment below or bring them up during the call. You can also share DevRel use cases, best practices, frameworks, and more by opening a new issue in the [GitHub Discussions](https://github.com/DevRel-Foundation/governance/blob/main/working-groups/themes.md)), which serves as the Foundation’s public forum. Additionally, we are always looking for 5-10 minute presentations in future community calls to spark discussion with the rest of the attendees and async.

### 🧹 Housekeeping
- Welcome
  The Developer Relations Foundation mission:

  > The mission of the Developer Relations Foundation project is to elevate the professional practice of developer relations and increase awareness of it as a driver of business value.

  > Welcome, everyone, to our DevRel Foundation open meeting. We are thrilled to have you with us today. Your presence here is not just appreciated; it's essential to our mission.
  > 
  > At the DevRel Foundation, we believe in the power of collaboration and the importance of diverse perspectives.
  > 
  > Your feedback is invaluable to us as we strive to create a space that truly supports and enhances the developer relations practice.
  > 
  > We are committed to continuously evolving and improving, and your insights play a crucial role in shaping the direction of our efforts.
  > 
  > Whether it's through suggestions, constructive criticism, or new ideas, we are here to listen and adapt.
  > 
  > Thank you for being part of this journey with us. Let's work together to build a community that serves the needs and aspirations of everyone involved in developer relations.

- [Code of conduct](https://github.com/DevRel-Foundation/governance/blob/main/code_of_conduct.md)). 
Instances of abusive, harassing, or otherwise unacceptable behavior, please report at conduct@dev-rel.org
- LF Antitrust Policies. At meetings among actual or potential competitors, there is a risk that participants in those meetings may improperly disclose or discuss information in violation of the Antitrust Laws, or otherwise act in an anti-competitive manner.  To avoid this risk, DRF participants must adhere to [the policies available in conduct of meeting section](https://www.linuxfoundation.org/legal/antitrust-policy) when participating in DRF calls. To contact LF regarding matters addressed by this Antitrust Policy, please send an email to [legal@linuxfoundation.org](mailto:legal@linuxfoundation.org), and reference “Antitrust Policy” in the subject line.

### 🤔 Open Office Hour topics

Time to listen to the voices of community members, answer questions, and tackle blockers together. Topics can be related to:

- **Working Groups update**: Communication to request participation in working groups is due very shortly. Team is presently working on an FAQ and the request form. See [project status](https://github.com/orgs/DevRel-Foundation/projects/7/views/1?pane=issue&itemId=82202215&issue=DevRel-Foundation%7Cgovernance%7C97).
- **DRF Website**: Updates from the inital website architecture deployment: [https://discord.com/channels/1255563562449899573/1293234213490790513/130069700911982597](https://discord.com/channels/1255563562449899573/1293234213490790513/1300697009119825971)1  
We've got a basic site up, available to interested contributors to start fleshing out the content and refining the design:[ https://devrel-website.netlify.app/](https://devrel-website.netlify.app/) A lot of the content you'll see there is still from the TODO site (as template was pull from there) but we can replace it as this group defines devrel foundation wbsite content  working repo with the apache 2.0 source code to raise issues and open PR is here:[ https://github.com/DevRel-Foundation/website](https://github.com/DevRel-Foundation/website)
There's one new page related with the local ambassador idea discussed in past community calls that has been developed. take a look here:[ https://devrel-website.netlify.app/people-map/](https://devrel-website.netlify.app/people-map/)
- Assign Roles for next Call: moderator, scribe, crier volunteers
- Hey YOU! 🙋🏽‍♀️🙋🏽🙋🏻🙋🏼‍♀️🙋🏻 Whether you are new to the call or a regular participant, the community shapes the conversations happening in DevRel Foundation calls! Please add your discussion topics as a comment below or bring them up during the call. You can also share any DevRel use cases, best practices, frameworks, etc as a forum, by opening a new issue in the GH Discussions.

## Essential Information

* 🕛 Thursday October 30, 12 - 13 pm UTC
* 📝 [g.doc agenda](https://docs.google.com/document/d/1I5dX4RbvJyt4Z2dTWsS1BjmzRQV8RHOJ5klJ_tZa_eg/edit?usp=sharing)

| Frequency | Duration | Zoom link | Join Discord |
| -- | -- | -- | -- |
| bi-weekly | 1 hour | [Here](https://zoom.us/j/97622449945?pwd=K2eKlYuqZwaYkzuH9I5zzrErLYf44M.1) | [Join](https://discord.gg/Vc6Qytc9vE) |

**Comment by leggetter** [comment]
# Meeting Notes:

🎥 N/A

## Participants (4)

- Phil Leggetter
- Anita Ihuman
- Johannes Dienst
- Ram Iyengar

## Discussions

### Working Groups

* PL shared that the SC is behind schedule but to lookout for the communication shortly across:
  * Email
  * Discord
  * Social media

### Website

* AI keen to get involved.
* PL suggested messaging in the [#website channel](https://discord.com/channels/1255563562449899573/1293234213490790513) in Discord.

### Open discussion

* AI is generally very keen to get involved. Website, working groups, etc. Was hoping to see confirmation of the working groups.
* RI shared it is Diwali today
